workflow_scripts/test_models.py

```diff
@@ -6,17 +6,26 @@
 import subprocess
 import sys
 import test_utils
+import os
 
 
-def main():
-    parser = argparse.ArgumentParser(description='Test settings')
-    # default all: test by both onnx and onnxruntime
-    # if target is specified, only test by the specified one
-    parser.add_argument('--target', required=False, default='all', type=str,
-                        help='Test the model by which (onnx/onnxruntime)?',
-                        choices=['onnx', 'onnxruntime', 'all'])
-    args = parser.parse_args()
+tar_ext_name = '.tar.gz'
+onnx_ext_name = '.onnx'
+
 
+def get_all_models():
+    model_list = []
+    parent_dir = ["text", "vision"]
+    for directory in parent_dir:
```

Comment:
No need for a local variable here:
```suggestion
    for directory in ("text", "vision"):
```